.github/workflows/docker.yml

```
@@ -8,11 +8,11 @@ jobs:
 
   docker:
     name: build and push
-    runs-on: ubuntu-latest
+    runs-on: s1tbx-latest
```

```suggestion
    runs-on: ubuntu-latest
```

Bringing back original name.